index.js

```diff
@@ -1,7 +1,14 @@
-require("./lib/profiler")();
```

Comment:
If we change this part we will probably miss some parts of the profile

Reply:
it will cause #7 -- generate the profile as the module is required:

```js
const CpuProfilerWebpackPlugin = require('cpuprofile-webpack-plugin');

module.exports = {
plugins: [// the plugin is not `newed` ]
}

```

Reply:
but this means we will not be able to profile the cli or anything else which happens before module creation

Reply:
the idea here is to collect as much data as possible

Reply:
I see, some users (like me) could be very confused!  better to document the behavior.